schema.sql

```diff
@@ -5,5 +5,6 @@ CREATE TABLE animals (
     date_of_birth DATE,
     escape_attempts INT,
     neutered BOOLEAN,
-    weight_kg DECIMAL
+    weight_kg DECIMAL,
+    species TEXT
```

Comment:
- [ ] This would technically work, but once the table is created you won't be able to do this, so the idea is to use the `ALTER TABLE` command, I would suggest using it as this is a better practice.